### ISSUE TITLE
[FEATURE] Be able to set the disk size of the Agent/Manager form the UI

### ISSUE THREAD

**Issue by Raimund58** [state-of-the-art/BlendNet#141]
### Is your feature request related to a problem? Please describe.
If I start 20 agents I end up with 4000GB of diskspace even if my .blend is only 500mb.

### Describe the solution you'd like
I would like to be able set the size of the disk. 
### Describe alternatives you've considered
Currently I change the code in `\2.91\scripts\addons\blendnet\BlendNet\providers\gcp\__init__.py`

**Comment by rabits** [comment]
Agree, useful & simple to implement by using provider settings (still in dev in https://github.com/state-of-the-art/BlendNet/pull/137)
